addons/mail/static/tests/core/common/message.test.js

```diff
@@ -25,14 +26,13 @@ test("following internal link from chatter does not open chat window", async fun
     await contains(".o_last_breadcrumb_item", { text: "Jeanne" });
     await click("a", { text: "Admin" });
     await contains(".o_last_breadcrumb_item", { text: "Mitchell Admin" });
-    /**
-     * Asserting 0 chat window at this step is not sufficient because it doesn't give enough time
-     * for the potential unwanted chat window to open. Opening another chat window and making sure
-     * only 1 is present at the end (rather than 2) makes the test more robust.
-     */
+    // Assert 0 chat windows not sufficient because not enough time for potential chat window opening.
+    // Let's open another chat window to give some time and assert only manually open chat window opens.
     await contains(".o-mail-ChatWindow", { count: 0 });
     await click(".o_menu_systray i[aria-label='Messages']");
     await click("button", { text: "New Message" });
-    await contains(".o-mail-ChatWindow-header", { text: "New message" });
+    await insertText("input[placeholder='Search a conversation']", "abc");
+    await click("a", { text: "Create Channel" });
+    await contains(".o-mail-ChatWindow-header", { text: "abc" });
```

Comment:
Extra chat window could just be created with proper pyEnv data with fold state instead of doing the flow.

Reply:
Test sucks: it tests a negative. I wouldn't mind to just drop it. The comment explains check no chat window is not reliable, but the extra flow doesn't even change when there's a chat window that pops in.

Reply:
It's not the best but it's better than nothing, especially when a positive is asserted first.